src/RestHandler.cpp

```diff
@@ -31,7 +31,7 @@ using web::http::status_codes;
 using web::http::experimental::listener::http_listener;
 using web::http::experimental::listener::http_listener_config;
 
-OBECA::RestHandler::RestHandler(const libconfig::Config& cfg, const std::string& url, const unsigned& total_cache_size, const std::map<std::string, std::unique_ptr<OBECA::Service>>& services )
+MBMS_RT::RestHandler::RestHandler(const libconfig::Config& cfg, const std::string& url, const unsigned& total_cache_size, const std::map<std::string, std::unique_ptr<MBMS_RT::Service>>& services )
```

Comment:
The config values need to be changed to `mw.` , for instance
`gw.http_server.cert`
`gw.http_server.key`